src/datasources/asset/components/AssetQueryEditor.test.tsx

```diff
@@ -32,14 +32,20 @@ it('renders with query defaults', async () => {
   render({} as AssetQuery)
   await workspacesLoaded()
 
+  expect(screen.getByRole('radio', { name: AssetQueryLabel.Metadata })).toBeChecked();
+  expect(screen.queryByLabelText('Group by')).not.toBeInTheDocument();
   expect(screen.getAllByRole('combobox')[0]).toHaveAccessibleDescription('Any workspace');
   expect(screen.getAllByRole('combobox')[1]).toHaveAccessibleDescription('Select systems');
 })
 
+
 it('renders with initial query and updates when user makes changes', async () => {
-  const [onChange] = render({ type: AssetQueryType.Metadata, minionIds: ['1'], workspace: '2' });
+  const [onChange] = render({ queryKind: AssetQueryType.Metadata, minionIds: ['1'], workspace: '2' } as AssetMetadataQuery);
   await workspacesLoaded();
 
+  // User selects metadata
```

Comment:
I would remove this comment or change it (and the one at line 81), as no action is triggered, it's just an expect